src/Scouting.elm

```diff
@@ -30,18 +31,25 @@ type alias Model =
     { scouter : String
     , team : Maybe Int
     , match : Maybe Int
-    , driverStation : String
+    , driverStation : String --PossibleStations

```

Comment:
???

Reply:
I think it was a comment for me to make a 'possible stations' list or something like that.. Deleted it now because I don't think it's relevant anymore.